### ISSUE TITLE
EROFS Error on Ubuntu

### ISSUE THREAD

**Issue by feelthatvib3** [drcika/apc-extension#127]
Hi, everyone.

Clear install of the `VSCode` with `Apc Customize UI++` gives the following error:

```
Apc Extension failed: Error: EROFS: read-only file system, rename '/snap/code/148/usr/share/code/resources/app/out/bootstrap-amd.js' -> '/snap/code/148/usr/share/code/resources/app/out/bootstrap-amd.js.apc.extension.backup'
```

**Comment by ghost** [comment]
Fix: https://github.com/drcika/apc-extension/issues/49#issuecomment-1646332813

**Comment by feelthatvib3** [comment]
> Fix: [#49 (comment)](https://github.com/drcika/apc-extension/issues/49#issuecomment-1646332813)

Hm. In my understanding 'claiming ownership of the installation directory' means going to the path `/snap/code/148/usr/share/code/resources/app/out` and run `sudo chown -R username:groupname .`. Correct me if I am being wrong here, but that's what I did and it didn't help, unfortunately. Let me know if you need some additional info to address the matter.

**Comment by feelthatvib3** [comment]
Redid my previous steps just with another way of installing Visual Studio Code. Instead of installing it from Ubuntu App Center,  I downloaded the `.deb` package from the official [website](https://code.visualstudio.com/download), unpackaged it and then claimed ownership of the directory it has been installed in. Thanks for your time, resolved.
